how-to-contribute-to-workshops/how-to-contribute-to-workshops.md

```diff
@@ -0,0 +1,60 @@
+* Images on slide
+* Code on slide
+* Speaker notes
+
+In these Markdown files, it is essential that you not only provide the literal text that will go on the slide, but also add any additional notes about formatting, organization, etc. so that when the files are to be converted to slides, it is clear to **anyone** exactly how the slide should be formatted. For example, in the Markdown file, bold words that should be bolded on the slide and indicate if a certain text should be highlighted, underlined, etc. in the final product.
+
+For details about placing your work in GitBook, see <a href = "https://about.bitproject.org/teams/developer-relations/joining-as-an-organization#placing-your-work-in-gitbook">Placing your work in GitBook</a>.
+
+## Adding a Workshop in Collaboration with a Company
+
+If you have a workshop in collaboration with a company that you would like to contribute, convert the content of the company's work into Markdown files, where a single Markdown file corresponds to a single slide of the presentation. See <a href = "#Porting">Porting an Existing Workshop Into Our System</a> for a general outline of how to create the Markdown files. Make sure to provide adequate credit to the company.
+
+Ensure that these presentations align with the requirements outlined in <a href = "https://about.bitproject.org/teams/developer-relations/joining-as-an-organization">Joining as an Organization</a>. To integrate the work into our system, see <a href = "https://about.bitproject.org/teams/developer-relations/joining-as-an-organization#placing-your-work-in-gitbook">Placing your work in GitBook</a>. 
+
+## Developing Workshops From Existing Activities/Labs
+
+In order to convert our existing activities/labs into workshops, you should still follow the guidelines outlined in <a href = "https://about.bitproject.org/teams/developer-relations/joining-as-an-organization">Joining as an Organization</a>. The following are additional recommendations for how you can effectively convert an activity or lab into a workshop.
+
+* Do NOT blindly copy/paste the information from the activity or lab to use in the presentation. This will often lead to an overwhelming amount of text on the screen. Instead, summarize the main points on the Markdown file corresponding to each slide and use speaker notes to fill in more detailed information that the presenter will explain to the audience.
+* Use plenty of images that presenters can use to explain concepts, helping minimize word count on each respective Markdown file/slide. These images do not necessarily need to be a part of the activity or lab themselves, but they should be related to the content on the slide.
+* Integrate code snippets as neccessary.
+* Try not to have more than one card from an activity/lab represented on a single Markdown file (which corresponds with a slide). If you find yourself doing this, it probably means that you are including too much text or not adequately presenting each card.
+
```

Comment:
I think you could add the link to the documentation I created about developing workshops from cards. Then the bullet point can be shorter and more brief.